README.md

```diff
@@ -82,9 +82,9 @@ To open an issue in this repository, follow these steps:
 3.  Select the appropriate template based on your needs:
     ![](https://i.imgur.com/34mZ1K0.png)
 
-        - [Bug Report](https://github.com/FoonieMagus/ApeironBattleDemo_IssueList/issues/new?assignees=&labels=bug&template=bug_report.yml&title=%5B%F0%9F%90%9B+Bug+Report%5D%3A+): For reporting any newly discovered bugs.
```

Comment:
Fixing alignment to show the correct content
Old:
![image](https://user-images.githubusercontent.com/64959125/216815710-7a3492f5-4881-49ac-979b-6e8c4f3c826b.png)
New:
![image](https://user-images.githubusercontent.com/64959125/216815736-1e4c6268-7133-4c8d-8002-a4c69de99257.png)